.github/workflows/client.yml

```diff
@@ -43,7 +44,7 @@ jobs:
           target: gobuild
           tags: go-build-env
           build-args: |
-          # REVISION=${{ github.sha }} TODO: set revision only when we want to publish the image
+            REVISION=${{ github.sha }}
           # VERSION= ? TODO: Configure version, sample: 1.7.6
```

Comment:
Actually, we don't need to configure `REVISION` and `VERSION` args for `go-build-env` image. We need to move it to the `Build and publish Docker Runtime Image` step.

Reply:
See 9fe88148564f31778bfe592e05c9916de4347f34